web/src/components/dashboard/BusinessStructurePrompt.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { BusinessStructurePrompt } from "@/components/dashboard/BusinessStructurePrompt";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import * as mockRouter from "@/test/mock/mockRouter";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+
+describe("<BusinessStructurePrompt />", () => {
+  const Config = getMergedConfig();
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRouter({});
+  });
+
+  it("hides business structure prompt button", () => {
+    render(<BusinessStructurePrompt isButtonHidden={true} />);
+    expect(screen.queryByTestId("business-structure-prompt-button")).not.toBeInTheDocument();
+  });
+
+  it("shows business structure prompt button", () => {
+    render(<BusinessStructurePrompt />);
+    expect(screen.getByTestId("business-structure-prompt-button")).toBeInTheDocument();
+  });
+
+  it("renders notCompletedTaskPromptAnyTask from config", () => {
+    useMockRouter({
+      asPath: "any",
+    });
+    render(<BusinessStructurePrompt />);
+    expect(screen.getByTestId("any-task-content")).toBeInTheDocument();
```

Comment:
This test isn't actually testing the config, which makes the test name a bit misleading. I'm fine with using the testid to make it easier to test for the markdown string, but maybe just changing the test name to match.